### PR TITLE
Fix hwameistor/resources/lvm-nodes in discussion #5226

### PR DESCRIPTION
**Pull request by WANG0608GitHub**
Fixed Discussion #5226

### CODE REVIEW

**Comment by WANG0608GitHub** [comment]
cc @SSmallMonster

**Comment by SSmallMonster** [comment]
This may not the root cause for failed claiming disk, already replied [there](https://github.com/DaoCloud/DaoCloud-docs/discussions/5226).

**Comment by SSmallMonster** [comment]
cc @jzhupup

File: docs/zh/docs/storage/hwameistor/resources/lvm-nodes.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ spec:
   nodeName: k8s-worker-4
   owner: local-storage
   description:
-    diskType: SSD
+    diskType: SSD # 需要根据节点磁盘实际情况做调整为 HDD 或者 NVMe
```
Comment (windsonsea) [inline docs/zh/docs/storage/hwameistor/resources/lvm-nodes.md:65]:
```suggestion
    diskType: SSD # 可能需要根据节点磁盘实际情况调整为 HDD 或者 NVMe
```

**Comment by windsonsea** [comment]
现在加了一条注释，应该可以做万能胶了：

```
diskType: SSD # 可能需要根据节点磁盘实际情况调整为 HDD 或者 NVMe
```

**Review by windsonsea — APPROVED** [review]

**Pull request merged**